packages/nodes-base/nodes/Anclap/actions/transactions/withdrawExchange/description.ts

```diff
@@ -168,13 +164,12 @@ export const withdrawExchangeDescription: INodeProperties[] = [
 		},
 		default: '',
 		description:
-			'Type of memo. One of text, id or hash. Deprecated because memos used to identify users of the same Stellar account should always be of type of id.',
+			'Type of memo. One of text, ID or hash. Deprecated because memos used to identify users of the same Stellar account should always be of type of ID.',
```

Comment:
```suggestion
			'Type of memo. One of text, ID or hash. Deprecated because memos used to identify users of the same Stellar account should always be of type of ID',
```